packages/replay/src/main.ts

```diff
@@ -394,7 +394,7 @@ async function doUploadRecording(
     recordingId,
   });
 
-  await client.uploadRecording(recording.path!, uploadLink, size);
+  await exponentialBackoffRetry(() => client.uploadRecording(recording.path!, uploadLink, size));
```

Comment:
Can you add a `debug()` entry within `client.uploadRecording()` so we have visibility into retries?

Reply:
yeah good call, I'll also record how long it took to fail, because I feel like that might tell us something.

Reply:
Looks like `debug` does the time offset automatically! ❤️